tests/test_commands/test_utils.py

```diff
@@ -0,0 +1,232 @@
+import pytest
+
+from etna.commands.utils import _estimate_n_folds
+from etna.commands.utils import _max_n_folds_backtest
+from etna.commands.utils import _max_n_folds_forecast
+from etna.commands.utils import estimate_max_n_folds
+from etna.metrics import MAE
+from etna.models import HoltWintersModel
+from etna.models import LinearPerSegmentModel
+from etna.models import SeasonalMovingAverageModel
+from etna.pipeline import Pipeline
+from etna.transforms import DensityOutliersTransform
+from etna.transforms import DifferencingTransform
+from etna.transforms import LagTransform
+from etna.transforms import MeanTransform
+
+
+def run_forecast_test(pipeline, context_size, ts, expected):
```

Comment:
From the name of the file it isn't clear that this test is made for `estimate_max_n_folds`. We should rename the file or the test.